dll/win32/shell32/dialogs/dialogs.cpp

```diff
@@ -1692,3 +1684,23 @@ void WINAPI ExitWindowsDialog(HWND hWndOwner)
 
     FreeLibrary(msginaDll);
 }
+
+static BOOL IsShutdownAllowed(VOID)
```

Comment:
If this helper is used only once, the function it calls could be placed directly where it is needed.
Unless you know you want to use it elsewhere in this file in the future?